administrator-documentation/moderne-dx/how-to-guides/configure-dx-organizations.md

```diff
@@ -12,14 +12,13 @@ This guide assumes that:
 * You have deployed Moderne DX in your environment
 * You have already [configured Moderne DX to connect to your Artifactory instance](configure-dx-with-artifactory-access.md)
 
-## Organizations service template and API
+## File based organization structure
 
-You have two main options for building this service. You can:
-
-1. (**Recommended**) Fork our [Organizations service template](https://github.com/moderneinc/moderne-organizations) and modify it to meet your needs. Please see the [README](https://github.com/moderneinc/moderne-organizations/blob/main/README.md) for how to spin this up quickly. It can be as simple as updating a CSV file.
-2. Build your own service that fulfills the [GraphQL contract](https://github.com/moderneinc/moderne-organizations/blob/main/src/main/resources/schema/moderne-organizations.graphqls) using any GraphQL stack (e.g., NodeJS, Rust, C#, etc.)
+The simplest way to achieve the organization structure is to supply a repos.csv file directly in DX.
 
-We generally recommend forking the template and modifying it as, in most cases, that will be faster and easier than building it yourself. Regardless of which one you choose, however, some developer time will be required on your end.
+Downsides of file based organization structure:
+- You cannot configure DevCenter
```

Comment:
I struggled to parse this a bit at first as I thought there were multiple DevCenters. I think we could fix this by just slightly rephrasing this and adding a link:

```suggestion
- You cannot configure any [DevCenter](/administrator-documentation/moderne-platform/how-to-guides/dev-center.md)
```